### PR TITLE
[MRG] backport hotfix from tbeu/matio#92

### PR DESCRIPTION
**Pull request by massich**
This is a hotfix to backport https://github.com/tbeu/matio/pull/92 while the changes are not in 1.5.12

### CODE REVIEW

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
@tbeu do you see any objection to merge this?

**Comment by tbeu** [comment]
Just go ahead.

**Comment by agramfort** [comment]
I don’t have commit rights

**Comment by tbeu** [comment]
Me neither.

**Comment by agramfort** [comment]
arfff. I guess we need to wait for @183amir

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,6 +7,8 @@ source:
   fn: matio-{{ version }}.tar.gz
   url: https://github.com/tbeu/matio/releases/download/v{{ version }}/matio-{{ version }}.tar.gz
   md5: 2968b5a95f05fab8b79ec87f9a6c0cac
+  patches:
+    - 0001-hotfix-backport-from-92.patch  # [linux]
 
 build:
   number: 0
```
Comment (183amir) [inline recipe/meta.yaml]: Please change the build number to 1 too

**Review by 183amir — CHANGES_REQUESTED** [review]

**Comment by 183amir** [comment]
@conda-forge-admin please rerender

**Comment by 183amir** [comment]
@conda-forge-admin, please rerender

**Comment by 183amir** [comment]
@conda-forge-admin, please rerender

**Review by 183amir — APPROVED** [review]

**Comment by 183amir** [comment]
Thank you.

**Pull request merged**